SIMS/SIMS/Repositories/SqlProductRepository.cs

```diff
@@ -0,0 +1,134 @@
+using System.Data;
+using Microsoft.Data.SqlClient;
+using SIMS.Models;
+
+namespace SIMS.Repositories;
+
+public class SqlProductRepository(string connectionString)
```

Comment:
lets add interface as well ISqlProductRepository
and maybee a better name for the interface will be IProductRepository As it's high level contract

Reply:
Done ✅